src/rpc/blockchain.cpp

```diff
@@ -1479,6 +1479,9 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
             "{\n"
             "  \"time\": xxxxx,        (numeric) The timestamp for the statistics in UNIX format.\n"
             "  \"txcount\": xxxxx,     (numeric) The total number of transactions in the chain up to that point.\n"
+            "  \"block_count\": xxxxx, (numeric) Size of the window in number of blocks.\n"
+            "  \"txratecount\": xxxxx, (numeric) The number of transactions in the window.\n"
+            "  \"txrateinterval\": xxxxx, (numeric) The elapsed time in the window.\n"
```

Comment:
I don't love the `txratecount` and `txrateinterval` names (rate and count are fundamentally different things, and `txrateinterval` mentions tx but not time, but the value returned is time).

I suggest:
- `window_block_count`
- `window_tx_count`
- `window_interval`

Note that new RPC arguments and return values should be snake case.